remote-access/samba.md

```diff
@@ -105,3 +105,25 @@ workgroup = <your workgroup name here>
 ```
 
 That should be enough to share the folder. On your Windows device, when you browse the network, the folder should appear and you should be able to connect to it.
+
+### "Host is down" error
+
+This error is caused by a combination of two things: A SMB protocol version mismatch, and the CIFS client on Linux returning a misleading error message. 
+
+In order to fix this a version entry needs to be added to the mount command. Note that by default the latest 'mount.cifs' on Raspberry Pi OS will attempt to negotiate the highest possible version greater or equal to 2.1 which should work with Windows 7 and above shares, but if connecting to older NAS devices for example, you may need to use an earlier version.
```

Comment:
These two paragraphs should be combined. Also, the last sentence is quite long and complex - what about this:

`By default Raspberry Pi OS will only use versions 2.1 or above, which is compatible with Windows 7 and later. Older devices, including some NAS, may require version 1.0: